keyboards/chavdai40/readme.md

```diff
@@ -0,0 +1,3 @@
+# chavdai40
```

Comment:
This file needs to follow the template, <https://github.com/qmk/qmk_firmware/blob/master/quantum/template/avr/readme.md>